plugins/faustwp/includes/replacement/callbacks.php

```diff
@@ -219,9 +219,10 @@ function post_link( $link ) {
  * @return string
  */
 function term_link( $term_link ) {
-	if ( ! is_rewrites_enabled() ) {
-		return $term_link;
-	}
+	if (
```

Comment:
@jasonbahl there's a syntax error here due to a missing `}`. This should fix it.

```suggestion
	if (
		! is_rewrites_enabled()
		|| ( function_exists( 'is_graphql_request' ) && is_graphql_request() )
	) {
		return equivalent_frontend_url( $term_link );
	}
```